src/esm_tools/cli.py

```diff
@@ -58,5 +58,37 @@ def create_new_config(name, type):
     return 0
 
 
+# TODO(PG): This doesn't belong in the cli.py but whatever...
```

Comment:
This is too echam-specific to be part of the ESM-Tools CLI, isn't it? We can discuss it on the monday meeting.